posts/2024-07-23_myg-10/README.md

```diff
@@ -0,0 +1,50 @@
+---
+publication_date: 2024-07-22T00:00:00Z
+slug: monthly-dev-10
+tags: [gnoland, ecosystem, updates, gnovm, tm2, test4, gnostudio, connect]
+authors: [Kouteki]
+---
+
+# The More You Gno 10: Test4 is Live!
+
+This edition focuses on Test4, a major milestone towards our mainnet. Test4 is the first true multinode testnet featuring DAOs and on-chain governance, offering a preview of what’s to come. Much excitement, such wow.
+
+We document everything in our weekly engineering updates and video recordings.
+
+# Gno Core Updates
+
+## Test4 is Live
+
+With 7 validator nodes running and 3 more about to be added via on-chain governance, Test4 is the first multinode testnet that has the complexity and the feature set we want for the gno.land mainnet. Find out more about it in [this article](https://www.gno.land/r/gnoland/blog:p/test4-live).
+
+## Changelog
+
+- Gnoweb live
```

Comment:
Would it be useful to add links to each of these first three bulleted items?

Reply:
One of the isn't publicly accessible, so I removed all three. Not feeling strongly either way.